src/EVault/modules/Initialize.sol

```diff
@@ -47,6 +47,14 @@ abstract contract InitializeModule is IInitialize, Base, BorrowUtils {
         vaultStorage.interestFee = DEFAULT_INTEREST_FEE.toConfigAmount();
         vaultStorage.creator = vaultStorage.governorAdmin = proxyCreator;
 
+        {
+            string memory underlyingSymbol = getTokenSymbol(address(asset));
+            uint256 seqId = sequenceRegistry.reserveSeqId(underlyingSymbol);
+
+            vaultStorage.symbol = string(abi.encodePacked("e", underlyingSymbol, "-", uintToString(seqId)));
```

Comment:
I think https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Strings.sol#L24 OZ version might be more gas optimized?